engine/source/materials/visc/visc_prony_lstrain.F

```diff
@@ -0,0 +1,357 @@
+Copyright>        but WITHOUT ANY WARRANTY; without even the implied warranty of
+Copyright>        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+Copyright>        GNU Affero General Public License for more details.
+Copyright>    
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>    
+Copyright>    
+Copyright>        Commercial Alternative: Altair Radioss Software 
+Copyright>    
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss 
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the 
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.    
+Chd|====================================================================
+Chd|  VISC_PRONY                    source/materials/visc/visc_prony.F
+Chd|-- called by -----------
+Chd|        VISCMAIN                      source/materials/visc/viscmain.F
+Chd|-- calls ---------------
+Chd|====================================================================
+      SUBROUTINE VISC_PRONY_LSTRAIN(
+     .           NEL     ,NUVAR   ,NPRONY  ,IADBUF  ,NUPARAM, UPARAM  ,
+     .           UVAR    , TIMESTEP  ,
+     .           MFXX    ,MFXY    ,MFXZ    ,MFYX    ,MFYY    ,MFYZ    ,
+     .           MFZX    ,MFZY    ,MFZZ    ,
+     .           S1      ,S2      ,S3      ,S4      ,S5      ,S6      )
+C----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+C---------+---------+---+---+--------------------------------------------
+C VAR     | SIZE    |TYP| RW| DEFINITION
+C---------+---------+---+---+--------------------------------------------
+C NEL     |  1      | I | R | SIZE OF THE ELEMENT GROUP NEL 
+C NUPARAM |  1      | I | R | SIZE OF THE USER PARAMETER ARRAY
+C NUVAR   |  1      | I | R | NUMBER OF USER ELEMENT VARIABLES
+C---------+---------+---+---+--------------------------------------------
+C TIMESTEP|  1      | F | R | CURRENT TIME STEP
+C UPARAM  | NUPARAM | F | R | USER MATERIAL PARAMETER ARRAY
+C RHO     | NEL     | F | R | INITIAL DENSITY
+C EPSPXX  | NEL     | F | R | STRAIN RATE XX
+C EPSPYY  | NEL     | F | R | STRAIN RATE YY
+C ...     |         |   |   |
+C ...     |         |   |   |
+C VISC    | NEL*6   | F | W | VISCOUS  STRESS 
+C ...     |         |   |   |
+C VISCMAX | NEL     | F | W | MAXIMUN DAMPING MODULUS(NEEDED FOR TIME STEP)
+C---------+---------+---+---+--------------------------------------------
+C UVAR    |NEL*NUVAR| F |R/W| USER ELEMENT VARIABLE ARRAY
+C---------+---------+---+---+--------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include      "param_c.inc"
```

Comment:
Hi @SaidMamouri 

It is no more allowed to use commons, can you use arguments instead ?

Thanks,
Sébastien